api/src/main/java/org/grnet/cat/api/endpoints/registry/MotivationEndpoint.java

```diff
@@ -1819,6 +1813,198 @@ public Response getCriteriaMetricsRelationship(@Parameter(
         return Response.ok().entity(response).build();
     }
 
+    @Tag(name = "Motivation")
+    @Operation(
+            summary = "Get a relationship between motivation, criterion and metric.",
+            description = "Get a relationship between motivation, criterion and metric.")
+    @APIResponse(
+            responseCode = "200",
+            description = "A relationship between motivation, criterion and metric.",
+            content = @Content(schema = @Schema(
+                    type = SchemaType.OBJECT,
+                    implementation = CriterionMetricResponseDto.class)))
+    @APIResponse(
+            responseCode = "401",
+            description = "User has not been authenticated.",
+            content = @Content(schema = @Schema(
+                    type = SchemaType.OBJECT,
+                    implementation = InformativeResponse.class)))
+    @APIResponse(
+            responseCode = "403",
+            description = "Not permitted.",
+            content = @Content(schema = @Schema(
+                    type = SchemaType.OBJECT,
+                    implementation = InformativeResponse.class)))
+    @APIResponse(
+            responseCode = "500",
+            description = "Internal Server Error.",
+            content = @Content(schema = @Schema(
+                    type = SchemaType.OBJECT,
+                    implementation = InformativeResponse.class)))
+    @SecurityRequirement(name = "Authentication")
+    @GET
+    @Path("/{id}/criteria/{criterion-id}/metrics")
+    @Produces(MediaType.APPLICATION_JSON)
+    public Response getCriterionMetricRelationship(@Parameter(
+            description = "The ID of the Motivation to get a relationship.",
+            required = true,
+            example = "pid_graph:3E109BBA",
+            schema = @Schema(type = SchemaType.STRING))
+                                                         @PathParam("id")
+                                                         @Valid
+                                                         @NotFoundEntity(repository = MotivationRepository.class, message = "There is no Motivation with the following id:")
```

Comment:
remove @CheckPublished, as currently it does not permit to get the metrics if the motivation is published